central/graphql/resolvers/image_vulnerabilities.go

```diff
@@ -707,3 +805,339 @@ func (resolver *imageCVEResolver) SuppressExpiry(_ context.Context) (*graphql.Ti
 func (resolver *imageCVEResolver) Suppressed(_ context.Context) bool {
 	return resolver.data.GetSnoozed()
 }
+
+// Following are the functions that return information that is nested in the CVEInfo object
+// or are convenience functions to allow time for UI to migrate to new naming schemes
+// This is awful
+func (resolver *imageCVEV2Resolver) ID(_ context.Context) graphql.ID {
```

Comment:
V2 versions of the resolver functions for simplicity. Old ones can be removed release after this goes live.